src/ui/pages/Notifications/NotificationItem.tsx

```diff
@@ -0,0 +1,84 @@
+import { IonItem, IonLabel, IonIcon } from "@ionic/react";
+import { ellipsisHorizontal } from "ionicons/icons";
+import i18next from "i18next";
+import KeriLogo from "../../assets/images/KeriGeneric.jpg";
+import { getConnectionsCache } from "../../../store/reducers/connectionsCache";
+import { useAppSelector } from "../../../store/hooks";
+import {
+  KeriaNotification,
+  NotificationRoute,
+} from "../../../core/agent/agent.types";
+import { i18n } from "../../../i18n";
+import { timeDifference } from "../../utils/formatters";
+
+const NotificationItem = ({
+  item,
+  index,
+  handleNotificationClick,
+}: {
+  item: KeriaNotification;
+  index: number;
+  handleNotificationClick: (value: string) => void;
+}) => {
+  const connectionsCache = useAppSelector(getConnectionsCache);
+
+  const notificationLabel = (item: KeriaNotification) => {
+    const connection = connectionsCache.filter(
+      (connection) => connection.id === item.connectionId
+    )[0]?.label;
+    // TODO: Implement different credential types
```

Comment:
Should we implement this now?

Reply:
At the moment I can't get that info from core because there is nothing in place to differentiate between one thing and another. I discussed that with Fergal and Thomas, this will need more refinement. For now I have only hardcoded one value (Driver's Licence) just to bring the message across and show it on the screen so the UI side is done, but the core part of that value still needs to be discussed and integrated.